util/logging/logging.go

```diff
@@ -23,16 +23,15 @@ func EventFields(event *types.Event) map[string]interface{} {
 	// adding the check name
 	if event.HasCheck() {
 		fields["check"] = event.Check.Name
-	} else if event.HasMetrics() {
-		fields["metric"] = true
 	}
 
 	if event.HasMetrics() {
-		fields["metrics"] = true
-	}
-
-	if event.HasMetrics() {
-		fields["metrics"] = true
+		count := len(event.Metrics.Points)
+		fields["metric count"] = count
```

Comment:
We should probably avoid whitespaces in the fields key, since it's harder to manipulate programatically. Instead `metric_count`, for example, would be easier!